tendrl/node_agent/objects/journal_mapping/__init__.py

```diff
@@ -0,0 +1,24 @@
+from tendrl.commons.etcdobj import EtcdObj
+from tendrl.commons import objects
+
+
+class JournalMapping(objects.BaseObject):
+    def __init__(self, node_id=None, storage_disks=None,
+                 unallocated_disks=None, *args, **kwargs):
+        super(JournalMapping, self).__init__(*args, **kwargs)
+
+        self.value = '_tmp/%s/journal_mapping/%s'
+        self.node_id = node_id
+        self.storage_disks = storage_disks
+        self.unallocated_disks = unallocated_disks
+        self._etcd_cls = _JournalMapping
+
+
+class _JournalMapping(EtcdObj):
+    __name__ = '_tmp/%s/journal_mapping/%s'
```

Comment:
@shtripat @r0h4n I think the output of this job should just be posted to etcd as the job output rather than any specific location. That way, the output would automatically be available to both the API and UI without any modifications required.